docs/tools/node_maintain.md

```diff
@@ -2,45 +2,60 @@
 
 ## Overview
 
-Graceful decommissioning could minimize the impact on existing job when removing unhealthy nodes. 
-This is a tool to help you achieve it in OpenPAI. 
+This [tool](../../src/tools/node_maintain.py) help you to gracefully decommission unhealthy nodes in OpenPAI. 
 
-In OpenPAI, an unhealthy node-list records the nodes to be decommissioned. 
-Components could sync the list and decommission them as needed.
-
-**NOTES: For now, only YARN nodes are decommissioned, HDFS doesn't.**
+**NOTES: This tool should be invoked in the dev-box, under `{PAI_ROOT_DIR}/src/tools`. For now, it only supports decommissioning YARN nodes.**
```

Comment:
So HDFS node was not decommissioned, suggest to point it out.

Reply:
Added